### DISCUSSION TITLE
Will there be a presentation session? When, now? What is the link?

### DISCUSSION THREAD

**Discussion by Garcia-INPE** [cu-esiil-edu/2024-data-short-course#88] [General]
@nquarder @eculler

**Comment by nquarder** [comment]
Hi Garcia! We had some time on Thursday for folks to share about their projects. We will also have time during open office hours beginning again in a couple weeks. Stay tuned for an announcement about that soon. Also, you could create a post here in the discussion and share your project with the community. That would be cool! :)

**Reply by Garcia-INPE** [reply]
Ok, @nquarder, I'll do that, my submission is almost complete, tks.
